_includes/releases/v22.2/v22.2.0-alpha.3.md

```diff
@@ -0,0 +1,99 @@
+- [`CREATE SCHEDULE`](../v22.2/create-schedule-for-backup.html) is no longer an admin only operation. Users should grant the appropriate [`BACKUP`](../v22.2/backup.html) privileges on the targets they wish to back up as part of the schedule. Cluster [backups](../v22.2/backup.html) require admin or system privilege `BACKUP`; DB backups require database privilege `BACKUP`; table backups require table privilege `BACKUP`. Release justification: high impact change for fine grained permission control. [#87188][#87188]
+- Decreased the cardinality of the number on `__moreN__` when replacing literals. [#87202][#87202]
+- When adding a `SECONDARY REGION` to a [multi-region database](../v22.2/multiregion-overview.html), the region is implicitly added to the regions list of the database if it was not present already. [#87108][#87108]
+- Added the `pg_get_function_def` [function](../v22.2/functions-and-operators.html), which returns the `CREATE` statement that can be used to create the given user-defined function. For built-in functions, it only returns the name of the function. Release justification: low risk enhancement for compatibility. [#87439][#87439]
+
+<h3 id="v22-2-0-alpha-3-operational-changes">Operational changes</h3>
+
+- This change introduces a new histogram implementation that will reduce the total number of buckets and standardize them across all usage. This should help increase the usability of histograms when exported to a UI (i.e., Grafana) and reduce the storage overhead. After applying this patch it is expected to see fewer buckets in prometheus/grafana, but still  have similar values for histogram percentiles due to the use of interpolated values by Prometheus. [#86671][#86671]
+
+<h3 id="v22-2-0-alpha-3-command-line-changes">Command-line changes</h3>
+
+- Improved the output of sinkless [changefeeds](../v22.2/create-changefeed.html) in the [`cockroach sql`](../v22.2/cockroach-sql.html) terminal. Release justification: Prevents UX degradation in v22.2 as a side effect of fixing the `bytea_output` bug. [#85181][#85181]
+
+<h3 id="v22-2-0-alpha-3-db-console-changes">DB Console changes</h3>
+
+- In the [session details](../v22.2/ui-sessions-page.html) page, users can click on a transaction fingerprint id from the list of cached transaction fingerprints to go to that transaction's details page. The app will also change the selected date range to that of the session's start (rounded down to the hour) and end time (rounded up to the hour) on click. [#86919][#86919]
+- Properly formatted the execution count under [Statement Details](../v22.2/ui-statements-page.html) page. Increased the timeout for Statement Details page, which now shows a proper timeout error when this happens, no longer crashing the page. [#87153][#87153]
+- Add a column selector to the statement insights page and add new contention, full scan, [transaction](../v22.2/transactions.html) id, transaction fingerprint id, and rows read/written info. [#87171][#87171]
```

Comment:
"Add" --> "Added"

Reply:
Changed this and other uses of "Add ..." to "Added ..." across the page